model-api-gen-gradle/src/functionalTest/resources/json/jetbrains.mps.baseLanguage.collections.json

```diff
@@ -1661,7 +1662,8 @@
             "properties": [
                 {
                     "uid": "83888646-71ce-4f1c-9c53-c54016f6ad4f/1240216724530/1240219919705",
-                    "name": "order"
+                    "name": "order",
+                    "type": "jetbrains.mps.baseLanguage.collections.structure.LinkedHashMapOrder"
```

Comment:
This shouldn't contain `.structure.`.  Use the existing function `fqName` in `MPSMetaModelExporter` (change the type of the parameter to `node<INamedStructureElement>`.